index.js

```diff
@@ -1 +1,56 @@
 // implement your API here
+const express = require('express');
+const db = require('./data/db')
+const server = express();
+server.use(express.json()); 
+
+server.post('/users', (req, res)=>{
```

Comment:
excellent job on your endpoint logic and using the correct http codes for the relevant situations.